includes/formatting.php

```diff
@@ -66,6 +66,10 @@ function get_body_contents( $content ) {
  * @return int The number of replacements made.
  */
 function sideload_images( $post_id ) {
+	require_once ABSPATH . 'wp-admin/includes/media.php';
+	require_once ABSPATH . 'wp-admin/includes/file.php';
+	require_once ABSPATH . 'wp-admin/includes/image.php';
+
 	$post    = get_post( $post_id );
 	$content = $post->post_content;
 	$pattern = '/["\'](https?:\/\/images.airstory.co\/[^"\']+)/i';
```

Comment:
Not a bad thought, especially if people are doing crazier things with the media. I have a feeling this will be more of a "as we get better use-cases, we find things to make smoother with the process"-type of item.